src/main/java/Meggy/Resource.java

```diff
@@ -0,0 +1,126 @@
+    public final static String KW_DUE = "by";
+    /** 'Start time' keyword of 'event' command. */
+    public final static String KW_STT = "from";
+    /** 'End time' keyword of 'event' command. */
+    public final static String KW_END = "to";
+    /** Notification message after 'todo/deadline/event' command. */
+    public final static String NOTIF_ADD = " Got it. Added this task:\n";
+    /** Notification message after 'list' command. */
+    public final static String NOTIF_LIST = " Here are the tasks in your list:\n";
+    /** Notification message after 'mark' command. */
+    public final static String NOTIF_MARK = " Booyah! Marked this task as done:\n";
+    /** Notification message after 'unmark' command. */
+    public final static String NOTIF_UNMK = " OK. Marked this task as not done:\n";
+    /** Notification message after syntax error. */
+    public final static String NOTIF_USAGE = " Usage: ";
+    /** Notification message after 'delete' command. */
+    public final static String NOTIF_DEL = " OK. Removed this task:\n";
+    /** Notification message after 'find' command. */
+    public final static String NOTIF_FIND = " Here are the matching tasks in your list:\n";
+    /** Front page wallpaper. */
+    public final static String MEGGY_LOGO = " __  __\n|  \\/  |\n| \\  / | ___  __ _  __ _ _   _\n| |\\/| |/ _ \\/ _`" +
+            " |/ _` | | | |\n| |  | |  __/ (_| | (_| | |_| |\n|_|  |_|\\___|\\__, |\\__, |\\__, |\n              __/ " +
+            "| __/ | __/ |\n             |___/ |___/ |___/\n";
+    /** Message header. */
+    public final static String MSG_HD = "------------------------------------------------------------\n";
+    /** Message tail. */
+    public final static String MSG_TL = MSG_HD + "\n>";
+    /** Front page greetings. */
+    public final static String GREETINGS = " Wommy! Get REKT by the upcoming star of Inkopolis, Meggy.Meggy!\n";
+    /** Notification message after 'bye' command. */
+    public final static String FAREWELL = " OK gotta go play more Turf Wars. Have a nice day!\n";
+    /** Icon for tasks marked 'done'. */
+    public final static char DONE_MK = 'X';
+
+    /** @deprecated Class stores all resource values statically should not be initialized. */
+    private Resource() {
+    }
+
+    /**
+     * Formats an array/list index number.
+     *
+     * @return Formatted index (starts with 1).
+     */
+    public static String idxFmt(int i) {
+        return " " + (i + 1) + '.';
+    }
+
+    /**
+     * Notification message after each change to task list.
+     *
+     * @return Message about task list size.
+     */
+    public static String nTaskFmt(int nTask) {
+        return " Now you have " + nTask + " task(s) in the list.\n";
+    }
+
+    /**
+     * Message of error caused by {@link NumberFormatException}
+     *
+     * @param arg Non-null. String in question.
+     * @return Error message.
+     */
+    public static String errNFE(String arg) {
```

Comment:
Perhaps it will be better to name this method as errNfe()?
```suggestion
    public static String errNfe(String arg) {
```